scope.md

```diff
@@ -29,7 +29,7 @@ Well, mostly. The description is pretty good, if you squint, but it does not fit
 
 - DIDComm doesn't always involve turn-taking and request-response.
 - DIDComm interactions can involve more than 2 parties, and the parties are not always individuals.
-- DIDComm may include formats other than JSON.
+- DIDComm may include formats other than JSON. {FTD: why make this expansion.  Seems to me if you focused the effort on building atop JWM you create a "waist" that will simplify discussions and speed adoption.   And yes, JWM could map to any number of underlying transport protocols (e.g. SMTP, HTTPS, etc)
```

Comment:
Agree with the comment, but let's move this to an issue for a more in depth discussion.

```suggestion
```